dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UsersServiceImpl.java

```diff
@@ -1107,6 +1070,52 @@ private String checkUserParams(String userName, String password, String email, S
         return msg;
     }
 
+    /**
+     * @return if check failed throw error, otherwise return null
+     */
+    private void checkUserParams(String userName, String userPassword, String email, String phone, Integer state) {
+
+        if (StringUtils.isNotEmpty(userName)) {
+
+            if (!CheckUtils.checkUserName(userName)) {
+                throw new ServiceException(String.format("userName %s doesn't valid", userName));
```

Comment:
same here, when we already have status, we can reuse the status.getMsg()

Reply:
```diff
- throw new ServiceException(String.format("userName %s doesn't valid", userName));
+ throw new ServiceException(Status.REQUEST_PARAMS_NOT_VALID_ERROR.getMsg(), userName));
```

Reply:
```diff
- throw new ServiceException(String.format("userName %s doesn't valid", userName));
+ throw new ServiceException(Status.REQUEST_PARAMS_NOT_VALID_ERROR.getMsg(), userName));
```

Reply:
> ```diff
> - throw new ServiceException(String.format("userName %s doesn't valid", userName));
> + throw new ServiceException(Status.REQUEST_PARAMS_NOT_VALID_ERROR.getMsg(), userName));
> ```

get